docs-web/src/test/java/com/sismics/docs/rest/TestReviewResource.java

```diff
@@ -0,0 +1,49 @@
+package com.sismics.docs.rest;
+
+import com.sismics.util.filter.TokenBasedSecurityFilter;
+import org.junit.Assert;
+import org.junit.Test;
+
+import javax.json.JsonObject;
+import javax.json.JsonValue;
+import javax.json.JsonWriter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Tests the ReviewResource.java file
+ *
+ * @author aruiz2
+ */
+
+public class TestReviewResource extends BaseJerseyTest {
+    private static final Logger log = LoggerFactory.getLogger(TestReviewResource.class);
+
+    /**
+     * Tests ReviewResource.java
+     */
+    @Test
+    public void testReviewResource() {
+        // Login admin
+        String adminToken = clientUtil.login("admin", "admin", false);
+        String username = "username";
+        String docId = "docId";
+        String averageRating = "avgRating";
+        JsonObject json = target().path("/user_rating").request()
+                .cookie(TokenBasedSecurityFilter.COOKIE_NAME, adminToken)
+                .get(JsonObject.class);
```

Comment:
Before testing content, test the length first

Reply:
Since you add stuff to the database and in the future we might too, should we leave the length check aside?